pallets/gear/Cargo.toml

```diff
@@ -177,27 +178,27 @@ std = [
 	"gear-runtime-interface/std",
 ]
 runtime-benchmarks = [
-    "frame-benchmarking/runtime-benchmarks",
-    "frame-system/runtime-benchmarks",
-    "frame-support/runtime-benchmarks",
-    "common/runtime-benchmarks",
-    "gear-core-backend/mock",
-    "gear-core-errors/codec",
-    "gear-sandbox",
-    "sp-consensus-slots",
-    "sp-consensus-babe",
-    "rand",
-    "rand_pcg",
-    "test-syscalls/wasm-wrapper",
-    "demo-read-big-state/wasm-wrapper",
-    "demo-proxy/wasm-wrapper",
-    "gsys",
-    "demo-reserve-gas/wasm-wrapper",
-    "demo-delayed-sender/wasm-wrapper",
-    "demo-constructor/wasm-wrapper",
-    "demo-waiter/wasm-wrapper",
-    "demo-init-wait/wasm-wrapper",
-    "demo-signal-entry/wasm-wrapper",
+	"frame-benchmarking/runtime-benchmarks",
```

Comment:
why

Reply:
Right now this file has both tabs and spaces as indentation. I thought we might wanna make this consistent at least across the file